src/runtime/hexagon_remote/halide_remote.cpp

```diff
@@ -113,20 +117,51 @@ typedef int (*set_runtime_t)(halide_malloc_t user_malloc,
 PipelineContext run_context(stack_alignment, stack_size);
 
 int halide_hexagon_remote_initialize_kernels(const unsigned char *code, int codeLen,
+                                             int use_dlopen, int use_dlopenbuf,
                                              handle_t *module_ptr) {
-    elf_t *lib = obj_dlopen_mem(code, codeLen);
-    if (!lib) {
-        log_printf("dlopen failed");
-        return -1;
-    }
+    log_printf("halide_hexagon_remote_initialize_kernels ->\n");
```

Comment:
These printfs are way too aggressive. log_printf has high overhead, we can't print except in error conditions, or protected by some kind of debug mechanism.